docs/microservices.json

```diff
@@ -307,25 +597,469 @@
             "version": null
         },
         {
-            "id": "fees-register-admin-web",
-            "name": "Fees Admin",
-            "group": "Fees",
-            "description": null,
-            "repository": null,
-            "spec": null,
+            "id": "snl-frontend",
+            "name": "SNL Frontend",
+            "group": "SNL",
+            "proType": "web",
+            "description": "",
+            "repository": "https://github.com/hmcts/snl-frontend",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/snl-frontend.json",
+            "dependencies": [
+                {
+                    "id": "snl-api",
+                    "hard": true,
+                    "apis": []
+                }
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "snl-events",
+            "name": "SNL Event",
+            "group": "SNL",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/snl-events",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/snl-events.json",
+            "dependencies": [
+                {
+                    "id": "snl-rules",
+                    "hard": true,
+                    "apis": []
+                }
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "snl-rules",
+            "name": "SNL Rules",
+            "group": "SNL",
+            "proType": "api",
+            "description": "",
+            "repository": "https://github.com/hmcts/snl-rules",
+            "spec": "https://hmcts.github.io/reform-api-docs/specs/snl-rules.json",
             "dependencies": [
+
+            ],
+            "apis": [],
+            "version": null
+        },
+        {
+            "id": "rpe-feature-toggle-web",
+            "name": "RPE Feature Toggle Web",
+            "group": "RPE",
+            "proType": "web",
+            "description": "",
+            "repository": "https://github.com/hmcts/ft-toggle-web",
```

Comment:
this project was a spike and had to be within hmcts to test out pipeline. Should be removed